src/db/mongo_operations.js

```diff
@@ -0,0 +1,42 @@
+import mongoose, { mongo } from "mongoose";
+
+const { Schema } = mongoose;
+
+export let BookReviewsModel, AuditLogModel;
+
+export function InitMongoConnection(mongoURL) {
+    if (
+        mongoose.connection.readyState == 1 ||
+        mongoose.connection.readyState == 2 ||
+        mongoose.connection.readyState == 3
+    ) {
+        return;
+    } else {
+        console.log(
+            "mongoose connection state",
+            mongoose.connection.readyState
+        );
+    }
+    const bookReviewSchema = new Schema({
+        bookid: String,
+        review: String,
+        rating: Number,
+    });
+    BookReviewsModel = mongoose.model("book_review", bookReviewSchema);
+
+    const auditLogSchema = new Schema({
+        log: Schema.Types.Mixed,
+        createdBy: String,
+        createdAt: Schema.Types.Date,
+        updatedBy: String,
+        updatedAt: Schema.Types.Date,
+    });
+    AuditLogModel = mongoose.model("audit_log", auditLogSchema);
```

Comment:
Separate files for each model and initialization would be better. A file that only needs to import BookReviewsModel becomes prone to code changes in AuditLogModel and initialization logic.